ci/sdk_go.go

```diff
@@ -85,8 +85,9 @@ func (t GoSDK) Publish(
 	githubToken *Secret,
 ) error {
 	return gitPublish(ctx, gitPublishOpts{
-		source:       "https://github.com/dagger/dagger.git",
-		alpineBase:   util.GoBase(t.Dagger.Source),
+		source: "https://github.com/dagger/dagger.git",
+		// FIXME: the go env is not alpine-based anymore, is that a problem?
+		alpineBase:   t.Dagger.Go().Env(),
```

Comment:
Ah, this is confusing. This should be renamed, it's the base container that the `sourceFilter` is run in.

So we could change this to `sourceContainer` or similar. `alpine` has nothing to do with it.

Reply:
If you're OK with that, I will leave the FIXME in place, and will address it in a follow-up PR (which I look forward to)

Reply:
Fixed this in a patch I pushed here.